demos/src/auto-refresh.mustache

```diff
@@ -0,0 +1,3 @@
+<div data-o-component="o-comments-count" data-o-comments-count-config-article-id="{{articleId}}" data-o-comments-count-config-auto-refresh="true">
```

Comment:
The `data-o-component` attribute must be set to the component name, which is just `o-comments`.  You could use another data attribute to decide whether the render the comments or the comment count.

Reply:
I was just thinking about how we could do this so I could add a suggestion. @ker-an and I will look into this